src/features/VisualNovel/pages/MediaPage/MediaPage.module.css

```diff
@@ -0,0 +1,11 @@
+.list {
+  display: flex; 
+  gap: 20px;
+  flex-wrap: wrap;
+}
+
+.image {
+  height: auto;
+  border-radius: 8px;
+  width: 320px;
+}
```

Comment:
Do you need `height: auto` property here? `img` and `video` tags have them by default